terraform/aws-constellation/variables.tf

```diff
@@ -0,0 +1,71 @@
+variable "name" {
+  type        = string
+  description = "Name of the Constellation cluster."
+}
+
+variable "image" {
+  type        = string
+  description = "Node image reference or semantical release version."
+} # is required resolve the AMI
```

Comment:
I think we can also put this comment in the description. Doesn't hurt to be verbose here.